Header header
float64 measurement_time
float32 velocity_latency
PointLLH position
Point3D position_std_dev
Point3D linear_velocity
Point3D linear_velocity_std_dev
int32 num_sats

int32 INVALID=0
int32 PROPAGATED=1
int32 SINGLE=2
int32 PSRDIFF=3
int32 PPP=4
int32 RTK_FLOAT=5
int32 RTK_INTEGER=6

int32 type
uint32 solution_status
uint32 position_type
